libs/components/packages/src/schematics/ng-generate/add-ai-skills/files/skills/skyux-test-driven-development/SKILL.md

```
@@ -0,0 +1,434 @@
+
+**Exceptions (ask your human partner):**
+
+- Throwaway prototypes
+- Generated code
+- Configuration files
+
+Thinking "skip TDD just this once"? Stop. That's rationalization.
+
+## The Iron Law
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)